<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Удаление записи</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Введите ID записи</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Delete_row">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Delete_button">
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    border-style: outset;
    border-width: 2px;
    border-color: gray;
    padding: 4px;

        background-color: #25F7F2;
	border-radius: 10px;


}

QPushButton:pressed {
    border-style: inset;
     background-color: #25F7A2;
}
</string>
       </property>
       <property name="text">
        <string>удалить запись</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Delete_error">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>5</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
